### PR TITLE
chore: remove code deps

### PR DESCRIPTION
**Pull request by simoesrf**

### CODE REVIEW

File: .eslintrc.cjs
Comment (bernardobelchior) [inline .eslintrc.cjs:1]: I'm not sure if all of these make sense. Should we just cherry-pick the rules that we want/base them on the recommended config?

Comment (ricardofelgueiras) [reply .eslintrc.cjs:1]: I've reduced the complexity of the file and kept only the recommended I've found on their official website.

File: src/components/loading-skeleton.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
+import React from 'react';
+
 function randomNumber(min: number, max: number) {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
 
-const LoadingSkeleton = () => {
+const LoadingSkeleton: React.FC = () => {
```
Comment (pf-costa) [inline src/components/loading-skeleton.tsx:7]: Is this repo just using Preact or is mixed with React?

Comment (ricardofelgueiras) [reply src/components/loading-skeleton.tsx:7]: Good point. My mistake. I'll replace with preact correspondent component.

**Pull request merged**